### PR TITLE
Hass0912

### PR DESCRIPTION
**Pull request by CoMPaTech**
Adding heating and operation mode status wrt https://github.com/laetificat/anna-ha/pull/1

### CODE REVIEW

**Comment by laetificat** [comment]
Thank you for your pull request, I'll check it out and test it asap

File: haanna/haanna.py [hunk 1]
```diff
@@ -84,6 +95,14 @@ def set_preset(root, preset):
 
         return r.text
 
+    @staticmethod
+    def get_heating_status(root):
+        """Gets the active heating status"""
+        if root.find("appliance[type='heater_central']/logs/point_log/period/measurement").text == 'on':
```
Comment (laetificat) [inline haanna/haanna.py]: Add an extra filter to point_log to make sure the correct data is returned.
```suggestion
        if root.find("appliance[type='heater_central']/logs/point_log[type='central_heating_state']/period/measurement").text == 'on':
```

File: haanna/haanna.py [hunk 2]
```diff
@@ -187,6 +216,18 @@ def get_preset_dictionary(root, rule_id):
             preset_dictionary[directive.attrib['preset']] = float(directive.find("then").attrib['setpoint'])
         return preset_dictionary
 
+    @staticmethod
+    def get_active_mode(root, schema_ids):
+        """Gets the mode from a (list of) rule id(s)"""
+        active=False
+        for schema_id in schema_ids:
+            print(schema_id)
```
Comment (laetificat) [inline haanna/haanna.py]: Remove the print statement

File: haanna/haanna.py [hunk 3]
```diff
@@ -187,6 +216,18 @@ def get_preset_dictionary(root, rule_id):
...
+            print(root.find("rule[@id='" + schema_id + "']/active").text)
```
Comment (laetificat) [inline haanna/haanna.py:225]: Remove the print statement

**Comment by CoMPaTech** [comment]
Changed accordingly

**Comment by CoMPaTech** [comment]
Tiny ping of remembrance - as per multiple users interested from https://community.home-assistant.io/t/advice-how-to-load-sensor-info-from-plugwise-anna-thermostat-web-interface-xml/87801/22

**Pull request merged**